backend/services/reaction.py

```
@@ -0,0 +1,77 @@
+    obj_id = like_data.post_id if like_data.post_id else like_data.comment_id
+    return db.query(model).filter(model.id == obj_id).first().like_count
+
+def build_comment_response(comment, db: Session, user: User):
+    replies = db.query(Comment).filter(Comment.parent_id == comment.id).all()
+    return {
+        "id": comment.id,
+        "content": comment.content,
+        "created_at": comment.created_at,
+        "user": serialize_user(comment.user),
+        "total_likes": len(comment.likes),
+        "user_liked": any(l.user_id == user.id for l in comment.likes),
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)